bluesky/bundlers.py

```diff
@@ -416,6 +434,7 @@ async def save(self, msg):
         Expected message object is::
 
             Msg('save')
+            Msg('save', {"key": "value"})
```

Comment:
This should be Msg('save', key=value), else {"key": "value"} is going to be obj (the only positional argument) not kwargs.